src/modules/70-pipeline/components/PipelineDiagram/Nodes/StepGroupNode/StepGroupNode.tsx

```diff
@@ -109,7 +109,20 @@ export function StepGroupNode(props: any): JSX.Element {
 
   React.useEffect(() => {
     if (executionNode?.data?.executionGraph) {
-      setRetryStepGroupStepsData(processExecutionDataV1(executionNode.data.executionGraph))
+      const items: Array<PipelineGraphState> = []
+      const rootNodeId = executionNode.data.executionGraph.rootNodeId as string
+      const nodeAdjacencyListMap = executionNode?.data?.executionGraph.nodeAdjacencyListMap
+      const nodeMap = executionNode?.data?.executionGraph.nodeMap
+      processGroupItem({
+        items,
+        id: rootNodeId,
+        isRollbackNext: false,
+        nodeAdjacencyListMap,
+        nodeMap,
+        rootNodes: [],
+        isNestedGroup: true
+      })
```

Comment:
Variable declarations are not needed.
```suggestion
 const { rootNodeId, nodeAdjacencyListMap, nodeMap } = executionNode.data.executionGraph

      processGroupItem({
        items,
        id: rootNodeId as string,
        isRollbackNext: false,
        nodeAdjacencyListMap,
        nodeMap,
        rootNodes: [],
        isNestedGroup: true
      })
```

Reply:
done